### PR TITLE
JS Code clean up

### PR DESCRIPTION
**Pull request by piq9117**
Reference [#1](https://github.com/ragulbalaji/MachineLearnsTicTacToe/issues/1)
I'll do more when I find out more about trainingdata.json. 😄

### CODE REVIEW

**Review by klanmiko — APPROVED** [review]
ok

File: app/duplicates.js [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-fs.writeFileSync("data/trainingdata.json",JSON.stringify(unique[0]));
-for (var i=1;i<unique.length;i++)
-{
-  fs.appendFileSync("data/trainingdata.json",JSON.stringify(unique[i]));
-}
\ No newline at end of file
+
+console.log('generation done')
+
+fs.writeFileSync('data/trainingdata.json', JSON.stringify(unique[0]))
```
Comment (klanmiko) [inline app/duplicates.js:33]: why is this not using Ramda's map function that you wrote?

Comment (klanmiko) [reply app/duplicates.js:33]: line possibly causes a duplicate in the output file, given at unique[0]

**Review by klanmiko — CHANGES_REQUESTED** [review]
Possible bug on line 34. Have you tested this? I'm gonna open up a ticket for unit testing

**Comment by klanmiko** [comment]
By the way, training data.json is the encoded game state data and the move taken by the player as one object, and is a list of such objects

File: app/duplicates.js (hunk 1 above)
Comment (piq9117) [reply app/duplicates.js:33]:
```
fs.writeFileSync("data/trainingdata.json",JSON.stringify(unique[0])); // I'm not sure what this is doing yet...
for (var i=1;i<unique.length;i++) // My change started from this line.
{
  fs.appendFileSync("data/trainingdata.json",JSON.stringify(unique[i]));
}
```

**Pull request closed without merging**